index.html

```diff
@@ -2686,6 +2686,27 @@ <h3>
             <a>Navigate</a> <var>browsing context</var> to <var>url</var>.
           </li>
         </ol>
+      </section>
+	  <section>
+        <h3>
+          Privacy consideration: <a>ShortcutItem.url</a> tracking
+        </h3>
+        <p>
+          It's conceivable that any <a>ShortcutItem.url</a> could be crafted to
+          indicate that the application was launched from outside the browser
+          (e.g., <code>"url": "/task/?from=homescreen"</code>).
+          This can be useful for analytics and possibly other customizations.
```

Comment:
I'm hesitant to suggest this... specially if some UAs or users decide to start chopping/changing them. I wouldn't consider these reliable at all.  

```suggestion
```